html/components/masthead.stories.js

```diff
@@ -980,7 +976,7 @@ export const extended = () => {
               >
                 <span
                   class="
-                    sprk-c-Dropdown__title
+                    sprk-c-Masthead__selector-dropdown-title
                     sprk-b-TypeBodyTwo
                     sprk-o-Stack__item
                     sprk-o-Stack__item--flex@xxs
```

Comment:
These 4 are always used together. Let's combine what makes sense. At least the TypeBodyTwo I think makes sense.

Reply:
![image](https://user-images.githubusercontent.com/4342363/111501912-bfffa100-871b-11eb-860c-88d109ecc2d7.png)

I agree that this is an opportunity to combine, but this is also the styles of TypeBodyTwo -- do we want to over complicate the selector for this? I would also need to make variables for them.

color, font-weight, and size already available variables, combining this class would mostly be about being able to customize TextCrop

Reply:
I can see both approaches working in some ways, combining them vs. leaving. Though, we should leave the stack styles under the stack class so we can manipulate those easily and have access to all Stack offers without having to recreate it all. I vote to leave this as it and leave typebodytwo as separate. My reasoning is that it doesn't hurt to have it the way it is now and it communicates that its following the typebodytwo design and this will ensure that they stay in sync in future if any style changes come to typebodytwo they will also come here.

Reply:
Works for me!